.github/workflows/build.yml

```
@@ -0,0 +1,243 @@
+
+jobs:
+  prepare:
+    runs-on: ubuntu-latest
+    outputs:
+    # Make the version accessible to other jobs
+      new_version: ${{ steps.versioning.outputs.version }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0 # Necessary for SemVer
```

YAMLlint (1.38.0)

[warning] too few spaces before comment: expected 2

(comments)